### PR TITLE
Code mirror

### PR DESCRIPTION
**Pull request by tin0d0**

### CODE REVIEW

File: project/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,36 +1,75 @@
...
+    /* Height / width / positioning can be customized for your use case.
+       For demo purposes, we make firepad fill the entire browser. */
+    #firepad-container {
+      width: 100%;
+      height: 100%;
+    }
+  </style>
+</head>
+
+<body onload="init()">
```
Comment (g-luo) [inline project/src/main/webapp/index.html]: This code should be in script.js on master

**Review by g-luo — COMMENTED** [review]
This PR looks really similar to what's on master; do we still want to merge?
Maybe we should wait for merge once we decide how to handle toggling languages.

File: project/src/main/webapp/index.html [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
     <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/mode/javascript/javascript.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/mode/python/python.js"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/codemirror.css" />
     <link rel="stylesheet" href="https://firepad.io/releases/v1.5.9/firepad.css" />
+    <link rel="stylesheet" href="https://codemirror.net/theme/ayu-dark.css" />
     <script src="https://firepad.io/releases/v1.5.9/firepad.min.js"></script>
-
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js"></script>
     <style>
```
Comment (max-friedman) [inline project/src/main/webapp/index.html]: Is there a particular reason that you have all this styling in the HTML file and not in a separate stylesheet?

File: project/src/main/webapp/index.html [hunk 3]
```diff
@@ -27,10 +28,103 @@
...
+    <div class="header"></div>
+    <div class="operations">
+      Theme: 
+      <select id="select">
+        <option selected>ayu-dark</option>
+      </select>
+    </div>
     <div id="firepad-container"></div>
+
+    <script>
```
Comment (max-friedman) [inline project/src/main/webapp/index.html:33]: Similar to my other comment, is there a particular reason that these scripts are not in another file?

**Review by max-friedman — COMMENTED** [review]
Just a couple qs but overall looks good

**Pull request merged**